tests/unit/utils/content.test.js

```diff
@@ -0,0 +1,21 @@
+import { isContentTooShort } from 'utils/content';
+
+describe('.isContentTooShort', () => {
```

Comment:
Para seguir o padrão do projeto:

```suggestion
describe('isContentTooShort', () => {
```